<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #f3f3f3, stop:1 #e5e5e5);
    font-family: "Segoe UI", system-ui;
}
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="margin">
    <number>16</number>
   </property>
   <item>
    <widget class="QLineEdit" name="display">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
       <weight>50</weight>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
QLineEdit {
    background: white;
    border: 1px solid #d0d0d0;
    border-radius: 8px;
    padding: 16px 20px;
    color: #323130;
    selection-background-color: #0078d4;
}
      </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="verticalSpacing">
      <number>8</number>
     </property>
     <!-- 第一行 -->
     <item row="0" column="0">
      <widget class="QPushButton" name="button7">
       <property name="text">
        <string>7</string>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
    background: white;
    border: 1px solid #d0d0d0;
    border-radius: 6px;
    color: #323130;
    font-size: 18px;
    font-weight: normal;
    min-height: 56px;
}
QPushButton:hover {
    background: #f8f8f8;
}
QPushButton:pressed {
    background: #f0f0f0;
}
        </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="button8">
       <property name="text">
        <string>8</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background: white; border: 1px solid #d0d0d0; border-radius: 6px; color: #323130; font-size: 18px; font-weight: normal; min-height: 56px; } QPushButton:hover { background: #f8f8f8; } QPushButton:pressed { background: #f0f0f0; }</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="button9">
       <property name="text">
        <string>9</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background: white; border: 1px solid #d0d0d0; border-radius: 6px; color: #323130; font-size: 18px; font-weight: normal; min-height: 56px; } QPushButton:hover { background: #f8f8f8; } QPushButton:pressed { background: #f0f0f0; }</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="buttonDivide">
       <property name="text">
        <string>÷</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background: #f8f8f8; border: 1px solid #d0d0d0; border-radius: 6px; color: #323130; font-size: 18px; font-weight: normal; min-height: 56px; } QPushButton:hover { background: #f0f0f0; } QPushButton:pressed { background: #e8e8e8; }</string>
       </property>
      </widget>
     </item>
     <!-- 第二行 -->
     <item row="1" column="0">
      <widget class="QPushButton" name="button4">
       <property name="text">
        <string>4</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background: white; border: 1px solid #d0d0d0; border-radius: 6px; color: #323130; font-size: 18px; font-weight: normal; min-height: 56px; } QPushButton:hover { background: #f8f8f8; } QPushButton:pressed { background: #f0f0f0; }</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="button5">
       <property name="text">
        <string>5</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background: white; border: 1px solid #d0d0d0; border-radius: 6px; color: #323130; font-size: 18px; font-weight: normal; min-height: 56px; } QPushButton:hover { background: #f8f8f8; } QPushButton:pressed { background: #f0f0f0; }</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="button6">
       <property name="text">
        <string>6</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background: white; border: 1px solid #d0d0d0; border-radius: 6px; color: #323130; font-size: 18px; font-weight: normal; min-height: 56px; } QPushButton:hover { background: #f8f8f8; } QPushButton:pressed { background: #f0f0f0; }</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="buttonMultiply">
       <property name="text">
        <string>×</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background: #f8f8f8; border: 1px solid #d0d0d0; border-radius: 6px; color: #323130; font-size: 18px; font-weight: normal; min-height: 56px; } QPushButton:hover { background: #f0f0f0; } QPushButton:pressed { background: #e8e8e8; }</string>
       </property>
      </widget>
     </item>
     <!-- 第三行 -->
     <item row="2" column="0">
      <widget class="QPushButton" name="button1">
       <property name="text">
        <string>1</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background: white; border: 1px solid #d0d0d0; border-radius: 6px; color: #323130; font-size: 18px; font-weight: normal; min-height: 56px; } QPushButton:hover { background: #f8f8f8; } QPushButton:pressed { background: #f0f0f0; }</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="button2">
       <property name="text">
        <string>2</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background: white; border: 1px solid #d0d0d0; border-radius: 6px; color: #323130; font-size: 18px; font-weight: normal; min-height: 56px; } QPushButton:hover { background: #f8f8f8; } QPushButton:pressed { background: #f0f0f0; }</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="button3">
       <property name="text">
        <string>3</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background: white; border: 1px solid #d0d0d0; border-radius: 6px; color: #323130; font-size: 18px; font-weight: normal; min-height: 56px; } QPushButton:hover { background: #f8f8f8; } QPushButton:pressed { background: #f0f0f0; }</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="buttonMinus">
       <property name="text">
        <string>−</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background: #f8f8f8; border: 1px solid #d0d0d0; border-radius: 6px; color: #323130; font-size: 18px; font-weight: normal; min-height: 56px; } QPushButton:hover { background: #f0f0f0; } QPushButton:pressed { background: #e8e8e8; }</string>
       </property>
      </widget>
     </item>
     <!-- 第四行 -->
     <item row="3" column="0">
      <widget class="QPushButton" name="button0">
       <property name="text">
        <string>0</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background: white; border: 1px solid #d0d0d0; border-radius: 6px; color: #323130; font-size: 18px; font-weight: normal; min-height: 56px; } QPushButton:hover { background: #f8f8f8; } QPushButton:pressed { background: #f0f0f0; }</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="buttonClear">
       <property name="text">
        <string>C</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background: #f8f8f8; border: 1px solid #d0d0d0; border-radius: 6px; color: #323130; font-size: 18px; font-weight: normal; min-height: 56px; } QPushButton:hover { background: #f0f0f0; } QPushButton:pressed { background: #e8e8e8; }</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="buttonEquals">
       <property name="text">
        <string>=</string>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
    background: #0078d4;
    border: 1px solid #0078d4;
    border-radius: 6px;
    color: white;
    font-size: 18px;
    font-weight: normal;
    min-height: 56px;
}
QPushButton:hover {
    background: #106ebe;
}
QPushButton:pressed {
    background: #005a9e;
}
        </string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="buttonPlus">
       <property name="text">
        <string>+</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background: #f8f8f8; border: 1px solid #d0d0d0; border-radius: 6px; color: #323130; font-size: 18px; font-weight: normal; min-height: 56px; } QPushButton:hover { background: #f0f0f0; } QPushButton:pressed { background: #e8e8e8; }</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>